recipes/qvina/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "qvina" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hadim/qvina/archive/cmake.zip
+  sha256: b11a6e089dd00a528bc5ff6b83f79045311ec806c199963287bc0939166659f9
+
+build:
+  skip: True  # [win or osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.12
+    - make
+    - boost-cpp >=1.69.0
+  host:
+    - boost-cpp >=1.69.0
+  run:
+    - boost-cpp >=1.69.0
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/vina  # [not win]
+    - test -f ${PREFIX}/bin/qvina2  # [not win]
+    - test -f ${PREFIX}/bin/qvinaw  # [not win]
+    - vina --help  # [not win]
+    - qvina2 --help  # [not win]
+    - qvinaw --help  # [not win]
```

Comment:
You are skipping windows so these selectors are not needed :wink:

Reply:
Can I leave them here since I would like to get windows to work eventually once its merged?

Reply:
sure